action.yml

```diff
@@ -36,6 +36,19 @@ inputs:
   google-service-account:
       description: Service account to be used when the 'google-workload-identity-provider' input is specified)
       required: false
+  setup-azure:
+    description: Setup Azure
+    required: false
+    default: 'false'
+  azure-client-id:
+    description: Azure Client ID to be used for Azure OIDC auth
+    required: false
+  azure-tenat-id:
+    description: AzureAD ID of the organization you are using
+    required: false
+  azure-subscription-id:
+    description: Subscrition ID of you are using
```

Comment:
also typo here

Reply:
Thanks review and sorry for my typo miss.
Fix this commit.

https://github.com/diggerhq/digger/pull/572/commits/3dd709d8d65a067e042e79d717312bcd780d483f